### PR TITLE
BULK VALIDATION description-missing

### PR DESCRIPTION
**Pull request by mSarts**
## Summary

Fixes 30 issues across 30 files.

Fixes no issue numbers.

### CODE REVIEW

File: dotnet-desktop-guide/framework/winforms/controls/numericupdown-control-windows-forms.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "NumericUpDown Control"
+description: Learn about the NumericUpDown control in Windows Forms, which like a combination of a text box and a pair of arrows that the user can click to adjust a value.
```
Comment (brittmsantos) [inline dotnet-desktop-guide/framework/winforms/controls/numericupdown-control-windows-forms.md:3]: Missed a word--I believe you meant "which looks like"

File: dotnet-desktop-guide/framework/winforms/controls/multithreading-in-windows-forms-controls.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Multithreading in Controls
+description: Learn about multithreading in Windows Forms Controls, for example making your UI more responsive by performing time-consuming operations on another thread.
```
Comment (brittmsantos) [inline dotnet-desktop-guide/framework/winforms/controls/multithreading-in-windows-forms-controls.md:3]: I'd replace "for example" with "such as" here; it flows better in terms of grammar.

File: dotnet-desktop-guide/framework/winforms/controls/numericupdown-control-windows-forms.md (hunk 1 above)
Comment (mSarts) [reply dotnet-desktop-guide/framework/winforms/controls/numericupdown-control-windows-forms.md]: Addressed.

File: dotnet-desktop-guide/framework/winforms/controls/multithreading-in-windows-forms-controls.md (hunk 2 above)
Comment (mSarts) [reply dotnet-desktop-guide/framework/winforms/controls/multithreading-in-windows-forms-controls.md]: Addressed.

**Comment by mSarts** [comment]
**Global effort to fix validation errors**

 @adegeo 

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Comment by mSarts** [comment]
@adegeo - I'm following up on this PR. Have you had a chance to review? cc: @v-emilypr

**Review by adegeo — APPROVED** [review]

**Pull request merged**